package.json

```diff
@@ -17,6 +17,8 @@
     "stylelint": "stylelint '**/*.css'",
     "stylelint:fix": "pnpm run stylelint -- --fix",
     "test:integration": "turbo test:integration --log-order=stream --",
+    "test:unit:lib": "pnpm --filter lib run test:unit",
```

Comment:
Exposes lib test scripts without turbo to improve DX

Reply:
This goes against the Turbo patterns. If it's the terminal UI that is the problem please look into using the UI flag instead: https://turbo.build/repo/docs/reference/run#--ui

Reply:
Updated.